osu.Game/Overlays/MusicController.cs

```diff
@@ -136,11 +136,20 @@ public bool TogglePause()
         }
 
         /// <summary>
-        /// Play the previous track.
+        /// Play the previous track or restart the current track if it's current time below 5000ms
```

Comment:
The xmldoc should not refer to a concrete value like this. Once you refactor out the constant you can `<see cref="RestartCutoffPoint" />` it.